### ISSUE TITLE
Provide Debian packages

### ISSUE THREAD

**Issue by awilkins** [Komodo/KomodoEdit#1986]
Sadly, the marvellous Mystic-Mirage has hung up his patching gauntlets and stopped making a PPA for Komodo Edit (with integrated Unity menu patch).

https://launchpad.net/%7Emystic-mirage/+archive/ubuntu/komodo-edit

> Sorry, guys! No more updates here. I'm not using Komodo Edit anymore and today I am not able to build a new version of Komodo Edit. And I have no time to solve this. It has become too hard to support this PPA.

And just as you added a feature I wanted (Mustache syntax)!

As he says : 

> Please, pay attention to all 3,600+ downloaders of this PPA.

Showing off Komodo Edit on my dev machine at work turns heads that were previously buried in SpaceMacs and various evil Mac-based editors. I really wouldn't like to use it without the menu integration either - so much faster to type <alt>-Split than go hunting in the menu (and I've not quite got used to the behaviour of the Commando yet, but for things you know are in the menu, it's less quick and intuitive than the Unity HUD). People even thought about buying licenses for IDE, since we use Docker so much.

Don't let it die on the vine and get stuck behind the bleeding edge! The period when the menu integration changed from a plugin to a patch was really frustrating for me (it took Mystic-Mirage a while to change over), I even *gasp* started flirting with other editors like Atom.

**Comment by Defman21** [comment]
The global menu will probably back in Komodo 11, there's a pull request for this.

@Naatan said something about deb/Ubuntu packages:

> As for a Ubuntu specific build: I cannot currently commit to this but we are always mindful of new distribution options. Unfortunately with distributing to a new platform comes creating new builds, maintaining new builds and supporting new builds. There's a lot that goes into it that you aren't aware of as an end-user. I say this for the benefit of anyone reading this, because I'm sure @Mystic_Mirage knows exactly what I'm talking about after having maintained a Komodo edit build for a couple of years.
So in short, we may yet have official Komodo IDE and Edit builds for Ubuntu, but as of right now I cannot commit to this. That said any Ubuntu user can download our Linux distribution of Komodo IDE/Edit and have it running in minutes. This distribution is pre-compiled, all you do is extractit and run it. It's not as convenient as apt-get, but it should be sufficient for any existing users.

**Comment by awilkins** [comment]
Yeah, that's good. Currently trying to get a build cooking on my box to see
how much work it is.

**Comment by Defman21** [comment]
[You don't even need to build Komodo yourself.](https://defman.me/blog/globalmenu-4-life/)

**Comment by awilkins** [comment]
For the time being, until ActiveState change to a newer Firefox build than 35. Which I presume will come with 11.

Would be nice to get the Debian packaging recipes in the native source tree though, rather than mucking about repackaging the tarballs.

**Comment by Naatan** [comment]
We'll use the forum post for discussing this further, I've already shared my views on the matter and have nothing further to add at this time. Other than to say that I'd be supportive of someone taking over the toch for Mystic-mirage. Whether that is us is to be determined, but likely would not happen until Komodo 11.
